.github/workflows/nightly.yml

```diff
@@ -81,17 +82,20 @@ jobs:
 
       - name: configure git
         run: |
-          git config --global user.name "action:deploy"
-          git config --global user.email "action:deploy@github.com"
+          git config --global user.name "action:deploy[bot]"
+          git config --global user.email "action:deploy[bot]@github.com"
 
       - name: remove previous nightly build
         run: |
           cd gh-pages
+          echo "INFO: gh-pages branch at $(git rev-parse HEAD) pre-rebase"
+          echo "Use git fetch <upstream> <commit> if you need to recover the branch"
+          echo ""
           EDITOR='sed -i "s/pick \([0-9a-z]* -nightly build-\)/drop \1/"' \
-            git rebase -i --keep-base
+            git rebase -i gh-pages.root
```

Comment:
This doesn't work for me, how about `git rebase -i --root`?

Reply:
Ah, should be `git rebase -i origin/gh-pages.root` (I created that branch)